pkg/app/piped/cloudprovider/lambda/function.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lambda
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/yaml"
+)
+
+type FunctionManifest struct {
+	Name string
+	u    *unstructured.Unstructured
```

Comment:
Not sure about the benefit of using this `Unstructured` in this case.
We can also explicitly declare all fields in this struct and load them as other configurations at the `config` package.

Reply:
Agree 👍 the previous implementation is just a mock, plz take another look 🙏